src/jit/inline.h

```diff
@@ -691,6 +701,8 @@ class InlineContext
     InlineObservation m_Observation;      // what lead to this inline
     int               m_CodeSizeEstimate; // in bytes * 10
     bool              m_Success;          // true if this was a successful inline
+    bool              m_Devirtualized;    // true if this was a devirtualized call
```

Comment:
Right now as I see these fields are used only for dump purposes, do we need them in release?

Reply:
Eventually we may want this in release to propagate back to the runtime. Since it's not much space (and we can compress further using bitfields) I think it's fine to always have this.